soh/soh/Enhancements/randomizer/context.h

```diff
@@ -103,6 +108,8 @@ class Context {
     std::shared_ptr<Settings> mSettings;
     std::shared_ptr<EntranceShuffler> mEntranceShuffler;
     std::shared_ptr<Dungeons> mDungeons;
+    std::shared_ptr<Logic> mLogic;
+    std::shared_ptr<SaveContext> mSaveContext;
```

Comment:
This isn't used anywhere that I can see, if it's for a future refactor I'd remove it from this pr.